lib/Distribution/Opensuse/Tumbleweed.pm

```diff
@@ -52,6 +52,7 @@ use YaST::SystemSettings::SystemSettingsController;
 use YaST::Firewall::FirewallController;
 use YaST::DNSServer::DNSServerController;
 use YaST::DNSServer::DNSServerSetupController;
+use YaST::Kdump::KdumpController;
```

Comment:
for not using Controllers, as we don't have at the moment different versions, we can just put this on the Test layer.